test/src/test/java/org/corfudb/integration/cluster/Harness/Node.java

```diff
@@ -0,0 +1,43 @@
+package org.corfudb.integration.cluster.Harness;
+
+import lombok.Getter;
+
+/**
+ * An abstraction for a corfu server/node.
+ * <p>
+ * <p>Created by maithem on 7/18/18.
+ */
+public class Node {
+
+    @Getter
+    final String address;
+
+    @Getter
+    final String clusterAddress;
+
+    @Getter
+    final String logPath;
+
+    final public Action shutdown;
```

Comment:
Another idea would be to make these Actions enums. You can then define all the actions in one place and easier to read the implementation in one place. Do it if you find it easier that way. I am ok either ways.